docs/src/main/asciidoc/hibernate-search-orm-elasticsearch.adoc

```diff
@@ -1015,6 +1015,88 @@ You can enable AWS request signing in Hibernate Search by adding a dedicated ext
 See link:{hibernate-search-orm-elasticsearch-aws-guide}#aws-configuration-reference[the documentation for the Hibernate Search ORM + Elasticsearch AWS extension]
 for more information.
 
+[[management]]
+== Management endpoint
+
+:extension-status: preview
+include::{includes}/extension-status.adoc[]
+
+Hibernate Search extension has a capability to reindex the data through the xref:./management-interface-reference.adoc[management interface].
+By default, this endpoint is not available. It can be enabled through configuration properties as shown below.
+
+[source,properties]
+----
+quarkus.management.enabled=true <1>
+quarkus.hibernate-search.management.enabled=true <2>
+----
+<1> Enable the xref:./management-interface-reference.adoc[management interface].
+<2> Enable Hibernate Search specific management endpoints.
+
+Once the management is enabled, data can be re-indexed via `/q/reindex`, where `/q` is the default management root path.
+Reindex path (`/reindex`) can be changed via configuration property as shown below.
+
+[source,properties]
+----
+quarkus.hibernate-search.management.root-path=custom-root-path <1>
+----
+<1> Use a custom `custom-root-path` path for reindex management endpoint.
+If the default management root path is used then the reindex path becomes `/q/custom-root-path`.
+
+This endpoint accepts `POST` requests with `application/json` content type only.
+All indexed entities will be re-indexed if an empty request body is submitted.
```

Comment:
Maybe add a simple example here? Just so that people don't think they need to put all the complicated configuration below.